napari/utils/perf/_trace_file.py

```diff
@@ -71,22 +84,33 @@ def write_event(self, event: PerfEvent) -> None:
         event : PerfEvent
             Event to write.
         """
+        category = "none" if event.category is None else event.category
+
         # Event type "X" denotes a completed event. Meaning we already
         # know the duration. The format wants times in micro-seconds.
         data = {
-            "pid": self.pid,
+            "pid": event.pid,
+            "tid": event.tid,
             "name": event.name,
-            "cat": event.category,
-            "ph": "X",
+            "cat": category,
+            "ph": event.type,
```

Comment:
Ah, this answers my questions above. What is `ph` short for and is there a better description we could use for our attribute?

Reply:
`ph` is `phase type` but in the spec they define it like this `ph: The event type.` so that's why we call it `event.type` is as good as `event.phase` but we could change it. The whole definition from their spec:

```
ph: The event type. This is a single character which changes depending on the type of event being output. The valid values are listed in the table below. We will discuss each phase type below.
```

Reply:
I changed it to `event.phase` I think that's much clearer. Both `event` and `type` are very overloaded so `event.type` was confusing. They do call it "event type" in places, but phase is the official name.